Tests/Tests/CPSSpec.swift

```diff
@@ -0,0 +1,128 @@
+import SwiftCheck
+import XCTest
+import Runes
+
+
+class CPSSpec: XCTestCase {
+    func testFunctor() {
+        // fmap id = id
+        property("identity law") <- forAll { (x: CPSOf<Int>) in
+            let lhs = id <^> x.getAsync
+            let rhs = x.getAsync
+
+            return lhs == rhs
+        }
+
+        // fmap (f . g) = (fmap f) . (fmap g)
+        property("function composition law") <- forAll { (o: CPSOf<Int>, fa: ArrowOf<Int, Int>, fb: ArrowOf<Int, Int>) in
+            let f = fa.getArrow
+            let g = fb.getArrow
+            let x = o.getAsync
+
+            let lhs = f • g <^> x
+            let rhs = (curry(<^>)(f) • curry(<^>)(g))(x)
+
+            return lhs == rhs
+        }
+    }
+
+    func testApplicative() {
```

Comment:
Function Body Length Violation: Function body should be span 40 lines or less: currently spans 41 lines (function_body_length)